docs/fetching-incidents/README.md

```diff
@@ -0,0 +1,59 @@
+# Fetching Incidents
```

Comment:
Lets keep naming conventions here, rename the file name to `fetching_incidents`

Reply:
We can. The name of the file however is part of the url and underscores are odd for urls.